pkg/cableengine/syncer/syncer.go

```diff
@@ -201,6 +203,20 @@ func (i *GatewaySyncer) generateGatewayObject() *v1.Gateway {
 	}
 
 	if connections != nil {
+		if i.healthcheck != nil {
+			for index := range *connections {
+				connection := &(*connections)[index]
+				latencyInfo := i.healthcheck.GetLatencyInfo(connection.Endpoint.Hostname)
```

Comment:
What if two endpoints share the same hostname on the remote side? (for example, two clusters configured equally on some cloud.. this could happen)

I believe we should probably index them by the IP address being pinged.

Reply:
Done.